CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # 2.87.x
+* Bump Roslyn to 5.0.0-2.25368.8 (PR: [#8431](https://github.com/dotnet/vscode-csharp/pull/8431))
+  * Simplify stale project handling in hot reload(PR: [#79386](https://github.com/dotnet/roslyn/pull/79386))
```

Comment:
Nit, but these are all missing spaces before the PR numbers.

Reply:
opened https://github.com/dotnet/roslyn-tools/pull/1493/files